libraries/src/main/java/com/foodie/kafka/service/KafKaTopicListeners.java

```diff
@@ -15,11 +15,11 @@ public void listenUserTopic(Object message) {
 
     @KafkaListener(topics = "foodie-users", groupId = "foodie-users-group")
     public void listenUsersTopic(Object message) {
-    log.info("Received message in foodie-users: " + message);
+        log.info("Received message in foodie-users: " + message);
```

Comment:
Please fix, this form of string concatination actually uses more resources

Reply:
Done